doc/source/cluster/kubernetes.rst

```diff
@@ -292,6 +297,20 @@ To delete a running Ray cluster, you can run the following command:
 
   kubectl delete -f ray/doc/kubernetes/ray-cluster.yaml
 
+.. _k8s-gpus:
+
+Using GPUs
+----------
+
+Using GPUs on Kubernetes requires some setup.
+Here's where to find the relevant documentation for `GKE`_, for `EKS`_, and for `AKS`_.
+
+The `Ray Docker Hub <https://hub.docker.com/r/rayproject/>`_ hosts CUDA-based images packaged with Ray for use in Kubernetes pods. 
+For example, the image ``rayproject/ray-ml:nightly-gpu`` is ideal for running GPU-based ML workloads with the most recent nightly build of Ray.
+Read :ref:`here<docker-images>` for further details on Ray images. 
+
+If you run into problems setting up GPUs for your Ray cluster on Kubernetes, please reach out to us at `<https://discuss.ray.io>`_. 
+
```

Comment:
Can you also mention that you need to explicitly change the resource requirements for the containers in your yaml?

Also, a snippet of relevant changes would be really helpful (which I think is only a couple lines of code)!